build/charts/antrea/values.yaml

```diff
@@ -67,6 +67,25 @@ ovs:
   # configuration).
   hwOffload: false
 
+secondaryNetworkOVS:
+  # -- Enable secondary network OVS configuration in Antrea.
+  enable: false
+  # -- Secondary network OVS integration bridge name.
+  ovsIntBrName: "br-secnet-int"
+  # -- Secondary network OVS transport bridge name.
+  ovsTransBrName: "br-secnet-trans"
```

Comment:
we should spell out "integration", "transport" and "bridge" name in the property names, for consistency with other parts of the configuration

Reply:
took care.